qutip/core/operators.py

```diff
@@ -471,6 +472,90 @@ def create(N, offset=0, *, dtype=None):
     return qdiags(data, -1, dtype=dtype)
 
 
+def fdestroy(N, site=0):
+    """
+    Fermionic destruction operator.
+    We use the Jordan-Wigner transformation,
+    making use of the so-called Jordan-Wigner ZZ..Z strings,
+    to construct this as follows (in Latex):
+    a_j = sigma_z^{otimes j}
+          otimes (frac{sigma_x + i sigma_y}{2})
+          otimes I^{otimes N-j-1}
+
+    Parameters
+    ----------
+    N : int
+        Dimension of Hilbert space.
+
+    site : int (default 0)
+        The site in Fock space to add a fermion to.
+        Corresponds to j in the above JW transform.
+
+    Returns
+    -------
+    oper : qobj
+        Qobj for destruction operator.
+
+    Examples
+    --------
+    >>> fdestroy(2) # doctest: +SKIP
+    Quantum object: dims=[[2 2], [2 2]], shape=(4, 4),
+    type='oper', isherm=False
+    Qobj data =
+    [[0. 0. 1. 0.]
+    [0. 0. 0. 1.]
+    [0. 0. 0. 0.]
+    [0. 0. 0. 0.]]
+    """
+    if site == 0:
+        return tensor([destroy(2), *([identity(2)]*(N-1))])
+    else:
+        if N-site-1 > 0:
```

Comment:
Add a sanity check that `site < N` for both functions.

Reply:
done, thank you!